### ISSUE TITLE
BIA Review of Established Agreements

### ISSUE THREAD

**Issue by gary-fallon** [department-of-veterans-affairs/va.gov-team#53094]
## Description
Table 1 does not appear complete, nor does it align with the IRP with regard to the listed ISA and MOU agreements.
- [x] Determine if VFS team applications require an ISA or MOU.

## Tasks
The following tasks must be reviewed and completed as necessary.
- [x] Update Table 1 with all known ISA and MOU agreements.

## Acceptance
The following tasks must be completed before this issue can be considered done.
- [x] Review the document for accuracy and completeness.

**Comment by little-oddball** [comment]
This ticket is a child of the epic below which has the BIA, etc.: https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055

**Comment by little-oddball** [comment]
@gary-fallon - from what I recall from our review of the BIA these would be items we would be able to leverage the Product List with (comments are stripped out of the document so can't reference those). 

@alyssagallion - if that is the case, then I believe Joe Tice (PM) and Peter Hill could be helpful here in identifying those items.

I would note there are most likely additional details we'll need some conversation with PSEC on.

**Comment by little-oddball** [comment]
@gary-fallon - My understanding is all VFS teams fall under the Platform ATO, unless if they obviously get their own. I am not aware of which may or may not... and that seems like a question, if we need answer, that should be posed to the COR.

So if I am understanding what you are saying, these would be VFS teams and the Minor Applications would be services we rely on (internal or external).

@alyssagallion - based on that I still believe we need to engage Joe Tice (PM) and Peter Hill (Product list / VFS) as well as Nate Peterson (PM) and Eric Boehs (External Services).

**Comment by gary-fallon** [comment]
@little-oddball if they fall under out VA.gov ATO, they need to be within the defined FISMA boundary and adhere to all of our controls. If they have their own ATO, they could inherit some of our controls. We will be requesting clarification from our COR and the ISO.

If VFS teams are part of the services that VA.gov provides and fall within our defined RTO, they could also be considered Minor Applications. It depends on how you define VFS in relation to our, and possibly their, ATO.

**Comment by alyssagallion** [comment]
Assigning this ticket to @jhouse-solvd per our Slack conversation and meeting today.

**Comment by gary-fallon** [comment]
This item might require additional user stories once the tasks have been reviewed.

**Comment by jhouse-solvd** [comment]
[X-Post for visbility](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055#issuecomment-1472607957)

**Comment by alyssagallion** [comment]
During our Gap Analysis, we realized that the System Security Plan (SSP) was incorrect. We have since updated it, presented it to the ISSO and Ray Wang is now reviewing the associated tickets he needs to approve. Once those are reviewed and approved, @ScottCutlip will continue the work on SSP. 

SSP is the dependency. Until approved, all BIA work is on hold.

**Comment by jhouse-solvd** [comment]
Please [see the parent epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055#issuecomment-1589591963) for a recent update. 

Moving to review.

**Comment by jhouse-solvd** [comment]
The most recent info has been included in the BIA. 

Please [see the parent epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055#issuecomment-1589591963) for future updates. 

Closing.
